soccer/SimFieldView.cpp

```diff
@@ -68,9 +69,17 @@ void SimFieldView::mousePressEvent(QMouseEvent* me) {
 }
 
 void SimFieldView::mouseMoveEvent(QMouseEvent* me) {
+    _posLabel->move(QPoint(me->pos().x() - 45, me->pos().y() + 17));
```

Comment:
Hi, thanks for the comment! I have read your suggestion and decide to use leaveEvent. So then I will not be using a QCursor. And I will not be able to move this line to FieldView, because I no longer have a reference to the mouse position without QCursor.